kl_conferences/migrations/0001_initial.py

```diff
@@ -17,8 +17,13 @@ class Migration(migrations.Migration):
             name='ServerNode',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('display_name', models.CharField(max_length=255)),
-                ('url', models.CharField(max_length=1024)),
+                ('display_name', models.CharField(max_length=255, unique=True)),
+                ('hostname', models.CharField(max_length=1024, unique=True)),
+                ('api_secret', models.CharField(max_length=64)),
+                ('region', models.CharField(choices=[('dolnoslaskie', 'dolnoslaskie'), ('slaskie', 'slaskie')], max_length=32, null=True)),
+                ('cpu_count', models.SmallIntegerField(null=True)),
+                ('load_5m', models.DecimalField(decimal_places=2, max_digits=6, null=True)),
```

Comment:
Prometheus?
We will have also 1 and 15m ;)

Reply:
After thinking a while core metrics used for picking servers would be better locally, we don't couple strictly with Prometheus. So ignore previous comment.